### PR TITLE
Query and update animals

### PR DESCRIPTION
**Pull request by devMahmoud**
## In this milestone I did:
* Add a column species of type string to your animals table. Modify your schema.sql file.
* Insert the following data:

        Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
        Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
        Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
        Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
        Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
        Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
        Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.

* Add `species` column to `animals` table.

Screenshots:

Screenshot from 2022-07-07 17-20-05

Screenshot from 2022-07-07 17-30-06

Screenshot from 2022-07-07 17-42-05

Screenshot from 2022-07-07 18-05-58

Screenshot from 2022-07-07 19-16-38

Screenshot from 2022-07-07 19-17-13

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -8,3 +8,5 @@ CREATE TABLE animals (
     neutered boolean,
     weight_kg decimal
 );
+
+ALTER TABLE animals ADD species varchar(50);
```
Comment (keneogu) [inline schema.sql:12]: The varchar limit for this project should be 255 not 50. Please fix this

File: data.sql [hunk 2]
```diff
@@ -63,3 +63,108 @@ VALUES (
...
+    escape_attempts,
+    neutered,
+    weight_kg
+)
+VALUES (
+    'Angemon',
+    'Jun-12-2005',
+    1,
+    true,
+    45
```
Comment (keneogu) [inline data.sql]: The weight_kg should be -45.0 not 45

File: data.sql [hunk 3]
```diff
@@ -63,3 +63,108 @@ VALUES (
...
+    escape_attempts,
+    neutered,
+    weight_kg
+)
+VALUES (
+    'Squirtle',
+    'Apr-02-1993',
+    3,
+    false,
+    12.13
```
Comment (keneogu) [inline data.sql]: This should be -12.13 not 12.13

File: data.sql [hunk 4]
```diff
@@ -63,3 +63,108 @@ VALUES (
...
+    escape_attempts,
+    neutered,
+    weight_kg
+)
+VALUES (
+    'Plantmon',
+    'Nov-15-2021',
+    2,
+    true,
+    5.7
```
Comment (keneogu) [inline data.sql]: This should be -5.7 not 5.7

File: data.sql [hunk 5]
```diff
@@ -63,3 +63,108 @@ VALUES (
...
+    escape_attempts,
+    neutered,
+    weight_kg
+)
+VALUES (
+    'Charmander',
+    'Feb-08-2020',
+    0,
+    false,
+    11
```
Comment (keneogu) [inline data.sql]: This should be -11.0 not 11

File: queries.sql [hunk 6]
```diff
@@ -15,3 +15,19 @@ SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name NOT IN ('Gabumon');
 
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```
Comment (keneogu) [inline queries.sql:18]: You did not add any of the transaction queries. Please add the transaction queries.

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @devMahmoud,

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 6 above)
Comment (devMahmoud) [reply queries.sql:18]: This is from the previous project

File: schema.sql (hunk 1 above)
Comment (devMahmoud) [reply schema.sql:12]: It is not in the requirements

**Comment by devMahmoud** [comment]
Screenshot from 2022-07-07 22-34-20

**Review by DeliceLydia — APPROVED** [review]
## Hi @devMahmoud, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**